tracer/src/Datadog.Trace.SourceGenerators/InstrumentationDefinitions/InstrumentationDefinitionsGenerator.cs

```diff
@@ -0,0 +1,771 @@
+                            attributeData.ApplicationSyntaxReference?.GetSyntax(),
+                            Constants.InstrumentAttributeProperties.IntegrationName));
+                }
+
+                if (methodName is null)
+                {
+                    hasMisconfiguredInput = true;
+                    diagnostics ??= new List<Diagnostic>();
+                    diagnostics.Add(
+                        MissingRequiredPropertyDiagnostic.Create(
+                            attributeData.ApplicationSyntaxReference?.GetSyntax(),
+                            Constants.InstrumentAttributeProperties.MethodName));
+                }
+
+                if (returnTypeName is null)
+                {
+                    hasMisconfiguredInput = true;
+                    diagnostics ??= new List<Diagnostic>();
+                    diagnostics.Add(
+                        MissingRequiredPropertyDiagnostic.Create(
+                            attributeData.ApplicationSyntaxReference?.GetSyntax(),
+                            Constants.InstrumentAttributeProperties.MethodName));
+                }
+
+                (ushort Major, ushort Minor, ushort Patch) minVersion = default;
+                (ushort Major, ushort Minor, ushort Patch) maxVersion = default;
+                if (minimumVersion is null)
+                {
+                    hasMisconfiguredInput = true;
+                    diagnostics ??= new List<Diagnostic>();
+                    diagnostics.Add(
+                        MissingRequiredPropertyDiagnostic.Create(
+                            attributeData.ApplicationSyntaxReference?.GetSyntax(),
+                            Constants.InstrumentAttributeProperties.MinimumVersion));
+                }
+                else if (!TryGetVersion(minimumVersion, ushort.MinValue, out minVersion))
+                {
+                    hasMisconfiguredInput = true;
+                    diagnostics ??= new List<Diagnostic>();
+                    diagnostics.Add(
+                        InvalidVersionFormatDiagnostic.Create(
+                            attributeData.ApplicationSyntaxReference?.GetSyntax()));
+                }
+
+                if (maximumVersion is null)
+                {
+                    hasMisconfiguredInput = true;
+                    diagnostics ??= new List<Diagnostic>();
+                    diagnostics.Add(
+                        MissingRequiredPropertyDiagnostic.Create(
+                            attributeData.ApplicationSyntaxReference?.GetSyntax(),
+                            Constants.InstrumentAttributeProperties.MaximumVersion));
+                }
+                else if (!TryGetVersion(maximumVersion, ushort.MaxValue, out maxVersion))
+                {
+                    hasMisconfiguredInput = true;
+                    diagnostics ??= new List<Diagnostic>();
+                    diagnostics.Add(
+                        InvalidVersionFormatDiagnostic.Create(
```

Comment:
Same comment for MaximumVersion

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2288/commits/bb06fe0e6040cc2707f475d33e03a3e300719ddf